programmers-coding-test/kimjiwon/greedy/1이될때까지.js

```diff
@@ -0,0 +1,15 @@
+function minNumberTimes(number, dividedNumber) {
+  let count = 0;
+  while (number != 1) {
+    if (number % dividedNumber) {
+      number--;
+      count++;
+    } else {
+      number /= dividedNumber;
+      count++;
+    }
+  }
+  return count;
+}
```

Comment:
삼항 연산으로 간추릴 수 있다.
minNumberTimes() => getMinNumberCount() 가 적절해보인다.

```suggestion 
function getMinNumberCount(N, K) {
  let cnt = 0;
  while (N !== 1) {
    N = N % K ? N - 1 : N / K;  // N이 K로 나눠지지 않으면 N에서 1을 빼고, 나눠지면 N / K를 수행하여 N에 대입한다.
    cnt++;
  }
  return cnt;
}
```

Reply:
삼항연산자는 코딩테스트말고는 사용하면 안되는건가요 Hoxy..?